cli/cli_fileshare.go

```diff
@@ -203,6 +207,28 @@ func (c *cmd) FileshareAutoCompletePeers(ctx *cli.Context) {
 	}
 }
 
+// FileshareAutoCompleteClear implements bash autocompletion for history clearing
+func (c *cmd) FileshareAutoCompleteClear(ctx *cli.Context) {
+	if ctx.NArg() == 0 {
+		fmt.Println("all\n<time_period>")
```

Comment:
I think `<time_period>` is not ok, specially for `zsh`. Since there the value appears into the autocompletion list and it can be selected.